.github/workflows/stale-issue-needs-info.yml

```diff
@@ -10,9 +10,8 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'Help us move this issue forward. Since it has no activity after 15 days of requesting more information, a bot is marking the issue as stale. Please add additional information as a comment or this issued will be closed in 5 days.'
-        close-issue-message: 'This issue was closed because more information was requested and there was no activity. If this is a bug report and still a problem, please supply the additional information requested and reopen the issue.'
+        stale-issue-message: 'Help us help you. This issue is being marked stale since it has no activity after 15 days of requesting more information. Please add info requested so we can help move the issue forward. Note: The triage policy is to close stale issues that need more info and no response after 2 weeks.'
```

Comment:
I think we should stick with the "Help us move this issue forward" language. It feels a bit more collaborative than "Help us help you" for some reason (that phrasing has always felt a bit more transactional to me).

Reply:
:+1:  Updated.